gatsby-config.js

```diff
@@ -0,0 +1,34 @@
+module.exports = {
+	siteMetadata: {
+		title: `Deriv.com`,
```

Comment:
Eventually, we should have different title & description for each page. so I think this can be removed.

Reply:
Yes I'll check how we can make this dynamic + include translation for the description